README.md

```diff
@@ -2,13 +2,12 @@
 
 [![GitHub Build Status](https://github.com/cisagov/publish-egress-ip-lambda/workflows/build/badge.svg)](https://github.com/cisagov/publish-egress-ip-lambda/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) GitHub
-[AWS Lambda](https://aws.amazon.com/lambda/) project using the Python runtimes
-started. This skeleton project contains [licensing information](LICENSE), as
-well as [pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
+This repository contains code to create a Lambda function that can scan a set
```

Comment:
Since this repository technically only creates a Lambda function artifact that can be used to create an actual Lambda functioin.

```suggestion
This repository contains the code for a Lambda function that can scan a set
```